api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2.java

```diff
@@ -0,0 +1,167 @@
+import io.strimzi.api.kafka.model.status.HasStatus;
+import io.strimzi.api.kafka.model.status.KafkaMirrorMaker2Status;
+import io.strimzi.crdgenerator.annotations.Crd;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import io.sundr.builder.annotations.Inline;
+import lombok.EqualsAndHashCode;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.unmodifiableList;
+
+@JsonDeserialize
+@Crd(
+        apiVersion = KafkaMirrorMaker2.CRD_API_VERSION,
+        spec = @Crd.Spec(
+                names = @Crd.Spec.Names(
+                        kind = KafkaMirrorMaker2.RESOURCE_KIND,
+                        plural = KafkaMirrorMaker2.RESOURCE_PLURAL,
+                        shortNames = {KafkaMirrorMaker2.SHORT_NAME}
+                ),
+                group = KafkaMirrorMaker2.RESOURCE_GROUP,
+                scope = KafkaMirrorMaker2.SCOPE,
+                version = KafkaMirrorMaker2.V1BETA1,
+                versions = {
+                        @Crd.Spec.Version(
+                                name = KafkaMirrorMaker2.V1BETA1,
+                                served = true,
+                                storage = true
+                        ),
+                        @Crd.Spec.Version(
+                                name = KafkaMirrorMaker2.V1ALPHA1,
+                                served = true,
+                                storage = false
+                        )
+                },
+                subresources = @Crd.Spec.Subresources(
+                        status = @Crd.Spec.Subresources.Status()
+                ),
+                additionalPrinterColumns = {
+                        @Crd.Spec.AdditionalPrinterColumn(
+                                name = "Desired replicas",
+                                description = "The desired number of Kafka Mirror Maker 2 replicas",
+                                jsonPath = ".spec.replicas",
+                                type = "integer"
+                        )
+                }
+        )
+)
+@Buildable(
+        editableEnabled = false, 
+        generateBuilderPackage = false, 
+        builderPackage = "io.fabric8.kubernetes.api.builder", 
+        inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done")
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "apiVersion", "kind", "metadata", "spec", "status" })
+@EqualsAndHashCode
+public class KafkaMirrorMaker2 extends CustomResource implements UnknownPropertyPreserving, HasStatus<KafkaMirrorMaker2Status> {
+
+    private static final long serialVersionUID = 1L;
+
+    public static final String SCOPE = "Namespaced";
+    public static final String V1ALPHA1 = "v1alpha1";
+    public static final String V1BETA1 = "v1beta1";
```

Comment:
I think that version is just `V1ALPHA1` for now. It's the first iteration of MM2 in Strimzi.

Reply:
Updated